### DISCUSSION TITLE
Fixing color generating utilities

### DISCUSSION THREAD

**Discussion by afeefuddin** [twbs/bootstrap#38584] [General]
This is discussion to fix color generating utilities #38569

**Comment by afeefuddin** [comment]
Hey everyone, I have some queries 
1) How can I access the cloned bootstrap in my local pc to work in the other file for testing purposes
2)Do I have to install any dependencies or external files for this
I am a newbie, need help

**Reply by julien-deramond** [reply]
As in most Open Source projects/repositories, you can find Contributing Guidelines directly in the repo. Ours is available [here](https://github.com/twbs/bootstrap/blob/main/.github/CONTRIBUTING.md); it includes a section that explains how you can clone the project locally.

Then, in our `README.md`, you can learn [how to run the documentation locally](https://github.com/twbs/bootstrap/tree/main#running-documentation-locally).

You can start with these elements. And then you'll have to understand the logic of the framework internally to be able to resolve https://github.com/twbs/bootstrap/issues/38569 (which might not be the easiest btw because the maintainers have not yet analyzed it).
